src/gwt/src/org/rstudio/studio/client/workbench/views/output/find/FindInFilesDialog.java

```diff
@@ -126,7 +126,8 @@ public native final int getResultsCount() /*-{
       }-*/;
 
       public native final void clearResultsCount() /*-{
-         this.resultsCount = 0;
+         if (this.resultsCount !== null)
```

Comment:
The exception was happening when `this` is `null`, so it would still happen here. I think we need to either (a) check `dialogState_` for null upstream, or (b) check `this` itself here.

Reply:
Ok, that makes sense. I added an upstream check to the only place it makes sense for dialogState_ to be null.